mvx/caip10.md

```diff
@@ -0,0 +1,59 @@
+---
+namespace-identifier: mvx-caip10
+title: MultiversX Namespace - Addresses
+author: MultiversX Team <contact@multiversx.com>
+discussions-to: https://github.com/ChainAgnostic/namespaces/issues/89
+status: Draft
+type: Standard
+created: 2023-09-11
+requires: ["CAIP-2", "CAIP-10"]
+---
+
+# CAIP-10
+
+_For context, see the [CAIP-10][] specification._
+
+## Rationale
+
+The MultiversX Address format is bech32, specified by the [BIP_0173][].
+The human-readable-part of the bech32 addresses is "erd" which means that the MultiversX address always starts with an `erd1` ("1" is a separator) e.g.: `erd1sea63y47u569ns3x5mqjf4vnygn9whkk7p6ry4rfpqyd6rd5addqyd9lf2`.
```

Comment:
Is 1 the separator only for mainnet, or are there addresses that begin `erdD` and `erdT` ?

Reply:
Similar to Bitcoin SegWit bech32 addresses, `1` is the separator. For the moment, the only valid HRP is `erd` so testnet and devnet adresses will keep the same format of `erd1`